server/data/masApiClient.ts

```diff
@@ -101,6 +101,20 @@ export default class MasApiClient extends RestClient {
     return this.get({ path: `/compliance/${crn}`, handle404: false })
   }
 
+  async searchUserCaseload(
+    username: string,
+    page: string,
+    sortBy: string,
+    body: Record<never, never> = {},
+  ): Promise<UserCaseload> {
+    let pageQuery = '?size=10'
+    if (page) {
+      pageQuery = `${pageQuery}&page=${page}`
+    }
+    pageQuery = `${pageQuery}&sortBy=${sortBy}`
```

Comment:
Might be worth looking at [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) for this
```suggestion
    const pageQuery = new URLSearchParams({ size: "10", page, sortBy }).toString();
```